src/main/java/bio/terra/workspace/service/job/exception/DuplicateJobIdException.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.service.job.exception;
+
+import bio.terra.workspace.common.exception.ConflictException;
+
+/** An exception indicating a jobId is already in use. Error code is 409 CONFLICT. */
+public class DuplicateJobIdException extends ConflictException {
+  public DuplicateJobIdException(String message, Throwable cause) {
```

Comment:
My inclination is to add them as needed rather than copying unused boilerplate, but that's not a particularly strong opinion and isn't consistent throughout WM